ToggleButton/ToggleButton.module.less

```diff
@@ -61,7 +61,11 @@
 }
 
 .button {
+	display: inline-flex;
```

Comment:
Why did you add this line instead of `display: inline-block;`? As a component, it seems to be more natural to provide it with the `inline-block` for a button, a checkbox, a text and so on and the `block` for any container like a panel, popup, and so on.

Reply:
I modified the design. See next commit